cli/web_worker.rs

```diff
@@ -1,35 +1,21 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
-use crate::fmt_errors::JSError;
 use crate::ops;
 use crate::state::ThreadSafeState;
+use crate::worker::Worker;
 use crate::worker::WorkerChannels;
-use crate::worker::WorkerReceiver;
 use deno_core;
-use deno_core::Buf;
 use deno_core::ErrBox;
-use deno_core::ModuleSpecifier;
 use deno_core::StartupData;
 use futures::future::FutureExt;
-use futures::future::TryFutureExt;
-use futures::sink::SinkExt;
-use futures::task::AtomicWaker;
-use std::env;
 use std::future::Future;
+use std::ops::Deref;
+use std::ops::DerefMut;
 use std::pin::Pin;
-use std::sync::Arc;
-use std::sync::Mutex;
 use std::task::Context;
 use std::task::Poll;
-use tokio::sync::Mutex as AsyncMutex;
-use url::Url;
 
 #[derive(Clone)]
-pub struct WebWorker {
-  pub name: String,
-  pub isolate: Arc<AsyncMutex<Box<deno_core::EsIsolate>>>,
-  pub state: ThreadSafeState,
-  external_channels: Arc<Mutex<WorkerChannels>>,
-}
+pub struct WebWorker(Worker);
```

Comment:
I think "WebWorker" and "Worker" are a bit confusing. A triple slash comment here could illuminate the situation.

EG
```
/// WebWorker is wrapper around Worker. Worker is CLI's term of an Isolate. 
/// There are several types of workers: Main, Compiler, and WebWorker.
```